fedn/fedn/network/api/client.py

```diff
@@ -18,26 +19,36 @@ class APIClient:
     :type verify: bool
     """
 
-    def __init__(self, host, port, secure=False, verify=False):
+    def __init__(self, host, port=None, secure=False, verify=False, token=None, verbosity="INFO"):
         self.host = host
         self.port = port
         self.secure = secure
         self.verify = verify
+        self.header = {}
+        # Check if auth token is set by environment variable
+        env_token = os.environ.get("FEDN_AUTH_TOKEN", False)
+        if env_token:
+            self.header = {"Authorization": "Token {}".format(token)}
```

Comment:
The auth scheme should also be configurable. e.g via env var FEDN_AUTH_SCHEME, where default can be "Token" but for JWTs its "Bearer".